examples/random_walk.py

```diff
@@ -44,20 +32,25 @@
 FLAGS = flags.FLAGS
 
 
-def run_random_walk(env: CompilerEnv, step_count: int) -> List[float]:
+def run_random_walk(env: CompilerEnv, step_count: int) -> None:
     """Perform a random walk of the action space.
 
     :param env: The environment to use.
     :param step_count: The number of steps to run. This value is an upper bound -
         fewer steps will be performed if any of the actions lead the
         environment to end the episode.
-    :return: The list of observed rewards.
     """
-    rewards = []
+
+    "Use hashes of the environment to record its internal state"
```

Comment:
Nitpick:

```suggestion
    # Use hashes of the environment to record its internal state
```

(and for other comments in this file)

Reply:
Ignore this, this on an old diff! :)